acnsdk/src/main/java/com/arrow/acn/api/models/DeviceTypeTelemetryModel.java

```diff
@@ -54,24 +57,29 @@ public DeviceTypeTelemetryModel() {
     }
 
     protected DeviceTypeTelemetryModel(@NonNull Parcel in) {
-        controllable = in.readByte() != 0x00;
+        int size = in.readInt();
+        for(int i = 0; i < size; i++){
+            String key = in.readString();
+            String value = in.readString();
+            variables.put(key,value);
+        }
         description = in.readString();
         name = in.readString();
         type = in.readString();
     }
 
     /**
-     * @return The controllable
+     * @return The variables
      */
-    public boolean isControllable() {
-        return controllable;
+    public Map<String, String> isVariables() {

```

Comment:
probably it should be getVariables but not isVariables?

Reply:
sure!thx